sp800-63a/sec6_dc.md

```diff
@@ -2,46 +2,18 @@
 
 <div class="breaker"></div>
 
-## <a name="derived-authN"></a> 6. Derived Identity
+## <a name="derived-authN"></a> 6. Derived Credentials
 
-_This section is normative._
+_This section is informative._
 
-Derived identity (formerly known as derived credentials in previous versions of SP 800-63) is the process of an individual proving to a CSP that they are the rightful subject of an identity record (i.e., a credential) that is bound to one or more authenticators they possess. This process is made available by a CSP that wants individuals to have an opportunity to obtain new authenticators that are bound to the existing, identity proofed record, or credential. Since it is beneficial to the individual and the CSP to minimize the number of times the identity proofing process is repeated, deriving identity is accomplished by proving possession and successful authentication of the primary authenticator that is bound to the original, proofed digital identity.
+Derived credentials is the process of an individual proving to a CSP that they are the rightful subject of an identity record (i.e., a credential) that is bound to one or more authenticators they possess. This process is made available by a CSP that wants individuals to have an opportunity to obtain new authenticators that are bound to the existing, identity proofed record, or credential. Since it is beneficial to the individual and the CSP to minimize the number of times the identity proofing process is repeated, deriving identity is accomplished by proving possession and successful authentication of the primary authenticator that is bound to the original, proofed digital identity.
 
 The definition of derived in this section does *not* imply that an authenticator is cryptographically tied to a primary authenticator, for example deriving a key from another key.
 
 There are two specific use cases for deriving identity:
 
-1. A _claimant_ seeks to obtain a secondary authenticator, bound to a proofed identity record, for use only within the limits and authorizations of the primary authenticator.  For example, this could be a derived PIV credential in federal use cases. *This section covers this use case.*
+1. A _claimant_ seeks to obtain a derived PIV, bound to their identity record, for use only within the limits and authorizations of having a PIV smartcard. *This use case is covered in SP 800-157 Guidelines for Derived Personal Identity Verification (PIV) Credentials, [[SP 800-157]](#SP800-157)*. 
 2. An _applicant_ seeks to obtain an authenticator, bound to a proofed identity record, from a CSP with which the individual does not have a pre-existing relationship. For example, if an applicant wants to switch CSPs and/or have another authenticator at a new CSP for other uses (e.g. basic browsing vs. financial). *This use case is covered by allowable identity evidence in [Section 5.2](#validate).*
```

Comment:
I think this works.  Get the credential, and binding can be bring your own or issued, so we stay silent.  Nice.